src/templates/static/get_started/index.jsx

```diff
@@ -42,6 +43,70 @@ const CTA = () => (
     </div>
 );
 
+const VolatilityIndicesTable = () => {
+    const active_symbols = [
+        {
+            key             : 'r10',
```

Comment:
why do we need the key? why name it active symbols? it's about the RTP values

Reply:
When looping over components, we need to define a unique identifier for each component, (see `src/templates/static/get_started/index.jsx:97`). 

About the second question, I don't know I thought it is active symbols. renaming it to `rtp_values`